ipmof/energymap.py

```diff
@@ -6,19 +6,25 @@
 
 import xlrd
 import numpy as np
+import yaml
 
 from ipmof.forcefield import lorentz_berthelot_mix, lennard_jones
 from ipmof.crystal import MOF
 
 
-def energy_map(sim_par, mof, atom_list):
+def energy_map(sim_par, mof, atom_list, export=[False, os.getcwd()]):
```

Comment:
why not use two parameters, i.e. export=False, and simulation_dir=os.getcwd() ? putting them in an array just makes the code below more confusing when you refer to export[0] and export[1].

Also, a little odd to have your parameter default to the results of a method (i.e. os.getcwd()). Make sure you understand WHEN default parameters get evaluated. See http://docs.python-guide.org/en/latest/writing/gotchas/. If the user ever changes the cwd within python, that won't get reflected here.